### ISSUE TITLE
Confused about the size of the alternate and primary

### ISSUE THREAD

**Issue by wjwei-handsome** [chhylp123/hifiasm#94]
In my genome assembly project, I found that the size of  *.a_ctg.gfa is larger than *.p_ctg.gfa. According to my opinion, alternate assembly is the other side of the bubble in the primary, so why is it larger than the primary ??

**Comment by chhylp123** [comment]
I guess a_ctg.gfa has much more contigs than p_ctg.gfa, so that there might be more overlapped regions between contigs in a_ctg.gfa. Could you please check the non-overlapped size of a_ctg.gfa? Also, hifiasm enable purging in default so that it will also put some non-bubble regions in a_ctg. We will release a new version this day which can output assemblies in a better way.

**Comment by artorias111** [comment]
Hello! I have the same confusion, when I ran hifiasm without purging. Did you figure out why the alternate contig file is larger than the alternate contig file? Thanks.

**Comment by chhylp123** [comment]
Is the size of the primary assembly roughly equal to the estimated genome size?

**Comment by artorias111** [comment]
Not really, there's a difference of around 200Mb (the genome size estimate is around 1Gb)
